tests/integration/specs/visual-regression/visual-regression.spec.js

```diff
@@ -0,0 +1,132 @@
+describe('Visual regression tests', () => {
+  it('should verify snapshot of every story', () => {
+    expect(
+      Cypress.spec.name,
+      'Visual regression tests cannot run in All Specs mode because of bugs in Cypress.\nsee https://github.com/cypress-io/cypress/issues/3090.\n',
+    ).not.to.equal('All Specs');
+
+    Cypress.log({
+      displayName: 'INIT',
+      message: 'Obtaining all stories from storybook',
+    });
+    cy.visit('/');
+    cy.get('img[alt="Chameleon components"]', { log: false }).should((img) => {
```

Comment:
I find this intriguing! what's the motivation? Happy to keep it thought :smile:

Reply:
Yeah, this one is very tricky. The motivation is to wait until Storybook is fully initialized. But Storybook doesn't have `data-test` attributes and most of the classes on elements are generated from css-modules. So there is not much to use for selecting elements.

I found a "better" solution than checking the header logo has been loaded:
```
    // wait for storybook to initialize. initialization is done when left menu is loaded
    // we can check it by waiting on menu items.
    cy.get('a[id^=explore]');
```
^^ that will check the left menu items have been loaded, which means that Storybook is ready. It still bad when we compare it to approach via `data-test`, but that will do.